frontend/src/metabase/query_builder/components/AlertListPopoverContent.jsx

```diff
@@ -0,0 +1,254 @@
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import { getQuestionAlerts } from "metabase/query_builder/selectors";
+import { getUser } from "metabase/selectors/user";
+import { deleteAlert, unsubscribeFromAlert } from "metabase/alert/alert";
+import { AM_PM_OPTIONS, DAY_OF_WEEK_OPTIONS, HOUR_OPTIONS } from "metabase/components/SchedulePicker"
+import Icon from "metabase/components/Icon";
+import Modal from "metabase/components/Modal";
+import { CreateAlertModalContent, UpdateAlertModalContent } from "metabase/query_builder/components/AlertModals";
+import _ from "underscore";
+import cx from "classnames";
+import cxs from 'cxs';
+
+const customSpacing = cxs ({
+    marginLeft: '10px'
+})
+const ownAlertCustomSpacing = cxs ({
```

Comment:
I don't know that we need to specifically call this out as `customSpacing` in the name but It's not a huge deal to me either way.